### PR TITLE
Kutter restapi

### PR DESCRIPTION
**Pull request by Chufle**
Change log:
- variables.tf instead of locals.tf for use of variables for s3 bucket names
- new lambda in ./get_objects/src/get_objects.py to list objects in dynamodb kutter-table
- changed name of script for zip of lambda functions to ./infrastructure/scripts/make_zip_lampda_functions_src.sh
- changed deploy.yml because of lambda function zip script
- new file api_gateway.tf for creation of api gateway for lambda "get_objects"
- added policies for lambda function "get_objects" for dynamodb and logs (changed in files iam_policy_document.tf and iam_for_lambda.tf)

### CODE REVIEW

File: infrastructure/api_gateway.tf [hunk 1]
```diff
@@ -0,0 +1,58 @@
+resource "aws_api_gateway_rest_api" "get-objects-API" {
+  name        = "get-objects-API"
+  description = "for AWS Lambda trigger"
+}
+
+resource "aws_api_gateway_resource" "proxy" {
+  rest_api_id = "${aws_api_gateway_rest_api.get-objects-API.id}"
+  parent_id   = "${aws_api_gateway_rest_api.get-objects-API.root_resource_id}"
+  path_part   = "{proxy+}"
+}
```
Comment (fabianschmauder) [inline infrastructure/api_gateway.tf:10]: do you need the proxy? 
I think you can remove it

File: infrastructure/api_gateway.tf [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+resource "aws_api_gateway_method" "proxy" {
```
Comment (fabianschmauder) [inline infrastructure/api_gateway.tf:12]: maybe call it not proxy. maybe get kutter

**Review by fabianschmauder — COMMENTED** [review]
Great.
I just think you do not need the proxy path

File: infrastructure/api_gateway.tf (hunk 1 above)
Comment (Chufle) [reply infrastructure/api_gateway.tf:10]: true

File: infrastructure/api_gateway.tf (hunk 2 above)
Comment (Chufle) [reply infrastructure/api_gateway.tf:12]: I changed it to list-objects

**Pull request merged**